velox/exec/tests/UnnestTest.cpp

```diff
@@ -39,6 +40,19 @@ TEST_F(UnnestTest, basicArray) {
 
   auto op = PlanBuilder().values({vector}).unnest({"c0"}, {"c1"}).planNode();
   assertQuery(op, "SELECT c0, UNNEST(c1) FROM tmp WHERE c0 % 7 > 0");
+
+  // Tests with an array with dictionary encoded elements.
+  VectorFuzzer::Options opts;
+  opts.containerVariableLength = false;
```

Comment:
Why false? Usually, these kinds of tests have a loop with 100 or so iterations and allow Fuzzer to generate whatever encoding it chooses using fuzzInputRow.